pkg/csi/service/wcpguest/controller.go

```diff
@@ -66,6 +66,9 @@ var (
 	}
 )
 
+// Event.Reason for a failed pvc
+const pvcFailureReason = "ProvisioningFailed"
```

Comment:
Instead of defining a new const, can you import `"k8s.io/kubernetes/pkg/controller/volume/events"` and then access `ProvisioningFailed` const